.github/kubeval.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
should we configure github actions to run the script? (will it run automatically? what happens if it fails?)